translations/introduction.tex/fr.po

```diff
@@ -43,6 +46,9 @@ msgid ""
 "Magic III: The Board Game}} is a tactical strategy RPG board game for 1-3 "
 "players using the core box set, or more when using expansions."
 msgstr ""
+"\\hypertarget{Heroes of Might and Magic III}{\\textbf{Heroes of Might and "
+"Magic III: The Board Game}} est un jeu de stratégie pour 1 à 3 "
+"joueurs utilisant le set de base."
```

Comment:
I think the info about expansions is missing? If you're just copying from @Legrecofr's translation, it's not up-to-date.